core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -433,10 +433,13 @@ private void cacheTopicPartitionIds(TopicIdPartition topicIdPartition) {
      * @param partitionsBecomeLeader   partitions that have become leaders on this broker.
      * @param partitionsBecomeFollower partitions that have become followers on this broker.
      * @param topicIds                 topic name to topic id mappings.
```

Comment:
Also, add javadoc for `validateTasksForTieredStateOfTopic`

Reply:
Updated.